docs-source/content/userguide/cicd/why-layering-matters.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Why Layering Matters"
+date: 2019-04-11T13:27:58-04:00
+draft: false
+weight: 2
+description: "Discussion of why Docker image layering affects CI/CD processes."
+---
+
+
+#### How does layering affect our CI/CD process?
+
+Now that we know more about layering, let’s talk about why it is important to our 
+CI/CD process.  Let's consider the kinds of updates we might want to make to our domain:
+
+![Types of updates](/weblogic-kubernetes-operator/images/updates.png)
+
+You might want to update the domain by:
+
+- Installing a patch on the operating system or a library
+- Updating the version of the JDK you are using
+- Picking up a new version of WebLogic Server
+- Installing patches on WebLogic Server
+- Updating the domain configuration, for example:
+  - Adding or changing a resource like a data source or queue
+  - Installing or updating applications
+  - Changing various settings in the domain configuration
+
+If we just want to update the domain configuration itself, that is the top layer, 
+then it is pretty easy.  We can make the necessary changes and save a new version 
+of that layer, and then roll the domain.  We could also choose to just build another 
+layer on top of the existing top layer that contains our delta.  If the change is 
+small, then we will just end up with another small layer, and as we have seen, 
+the small layers are no problem. 
+
+But consider a more complicated scenario - let's take updating the JDK as an example 
+to understand the impact of layers.  Say we want to update from JDK 11.0.2 to 11.0.4 
```

Comment:
Instead of using JDK 11 in the example use JDK 8

Reply:
8u201 and 8u202 can be an example

Reply:
done